tests/test_type_stub.py

```diff
@@ -8,6 +8,7 @@
 
 
 @pytest.mark.codegen_required
+@pytest.mark.fluent_version("==24.1")
```

Comment:
@mkundu1, latest should be 24.1? or is it in a different tag now?

Reply:
We will keep 24.1. Thank you.